esp/esp/program/modules/handlers/volunteersignup.py

```diff
@@ -81,13 +82,19 @@ def signup(self, request, tl, one, two, module, extra, prog):
         if hasattr(request.user, 'email'):
             form.load(request.user)
 
-        #   Override default appearance; template doesn't mind taking a string instead
-        context['form'] = form._html_output(
```

Comment:
can you delete the form's `_html_output` method now, or is that a django thing?

Reply:
It's a django thing